lib/hoge.go

```diff
@@ -1,6 +1,8 @@
 package lib
 
+import "github.com/crifff/private"
+
 //GetStr is test function
 func GetStr() string {
-	return "hello"
+	return private.Hello()
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: private
* [structcheck] unused struct field undeclared name: private
* [aligncheck] undeclared name: private
* [errcheck] error return value not checked (undeclared name: private)
* [unconvert] undeclared name: private
* [varcheck] unused global variable undeclared name: private

<!-- Comment by [SideCI](https://sideci.com) -->